frontend/src/routes/email/+server.ts

```diff
@@ -1,13 +1,14 @@
-import { render } from '$lib/email/emailRenderer.server';
-import { json } from '@sveltejs/kit';
-import type { RequestEvent } from './$types';
-import { componentMap, EmailTemplate, type EmailTemplateProps } from './emails';
+import {render} from '$lib/email/emailRenderer.server';
+import {json} from '@sveltejs/kit';
+import type {RequestEvent} from './$types';
+import {componentMap, EmailTemplate, type EmailTemplateProps} from './emails';
+import type {Component} from 'svelte';
 
 export async function POST(event: RequestEvent): Promise<Response> {
   const request = await event.request.json() as EmailTemplateProps;
-  const {template, ...props} = request;
-  const component = componentMap[template];
-  if (!component) throw new Error(`Invalid email template ${template}.}`);
+  const {...props} = request;
```

Comment:
Commit 023f83c4 changes this to no longer turn `request` into `...props` but just use `request` directly, since there's no gain from going through that object spread any longer.